Code/GraphMol/MolEnumerator/MolEnumerator.h

```diff
@@ -263,12 +315,15 @@ enumerate(const ROMol &mol, const std::vector<MolEnumeratorParams> &paramsList);
 \param maxPerOperation: the maximum number of molecules which an individual
 operation is allowed to generate
 
+\param enumerateStereo: whether to enumerate stereoisomers
```

Comment:
Since you added the stereo enumeration as another kind of `MolEnumeratorOp`, why is this parameter needed?